<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>KstChangeFileDialog</class>
  <widget class="QDialog" name="KstChangeFileDialog" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>488</width>
        <height>600</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Change Data File</string>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item>
        <widget class="QGroupBox" name="buttonGroup1" >
          <property name="title" >
            <string>Vector and Matrix Selection</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item row="0" column="0" >
              <layout class="QHBoxLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item>
                  <widget class="QPushButton" name="_allFromFile" >
                    <property name="text" >
                      <string>All &amp;From</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="KComboBox" name="_files" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>7</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item row="2" column="0" >
              <layout class="QGridLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item rowspan="5" row="0" column="0" colspan="1" >
                  <widget class="Q3ListBox" name="ChangeFileCurveList" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>7</hsizetype>
                        <vsizetype>7</vsizetype>
                        <horstretch>1</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="selectionMode" >
                      <enum>Q3ListBox::Extended</enum>
                    </property>
                    <property name="whatsThis" stdset="0" >
                      <string>The source file for the vectors selected in this list will be changed.</string>
                    </property>
                  </widget>
                </item>
                <item row="0" column="1" >
                  <widget class="QPushButton" name="ChangeFileClear" >
                    <property name="text" >
                      <string>C&amp;lear</string>
                    </property>
                    <property name="whatsThis" stdset="0" >
                      <string>Unselect all vectors in the list.</string>
                    </property>
                  </widget>
                </item>
                <item row="4" column="1" >
                  <spacer name="spacer2" >
                    <property name="sizeHint" >
                      <size>
                        <width>80</width>
                        <height>51</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Vertical</enum>
                    </property>
                  </spacer>
                </item>
                <item row="1" column="1" >
                  <widget class="QPushButton" name="ChangeFileSelectAll" >
                    <property name="text" >
                      <string>S&amp;elect All</string>
                    </property>
                    <property name="whatsThis" stdset="0" >
                      <string>Select all vectors in the list.</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
            <item row="1" column="0" >
              <layout class="QGridLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item row="0" column="0" >
                  <widget class="QPushButton" name="_clearFilter" >
                    <property name="text" >
                      <string/>
                    </property>
                  </widget>
                </item>
                <item row="0" column="1" >
                  <widget class="QLineEdit" name="_filter" >
                    <property name="sizePolicy" >
                      <sizepolicy>
                        <hsizetype>7</hsizetype>
                        <vsizetype>0</vsizetype>
                        <horstretch>1</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="QGroupBox" name="buttonGroup2" >
          <property name="title" >
            <string>New Data File</string>
          </property>
          <layout class="QGridLayout" >
            <property name="margin" >
              <number>0</number>
            </property>
            <item row="0" column="0" >
              <layout class="QGridLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item row="0" column="0" >
                  <widget class="QLabel" name="textLabel1" >
                    <property name="text" >
                      <string>&amp;File:</string>
                    </property>
                    <property name="buddy" stdset="0" >
                      <cstring>_dataFile</cstring>
                    </property>
                    <property name="wordWrap" >
                      <bool>false</bool>
                    </property>
                  </widget>
                </item>
                <item row="0" column="1" >
                  <layout class="QHBoxLayout" >
                    <property name="margin" >
                      <number>0</number>
                    </property>
                    <item>
                      <widget class="KUrlRequester" name="_dataFile" />
                    </item>
                  </layout>
                </item>
              </layout>
            </item>
            <item row="1" column="0" >
              <layout class="QGridLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item row="0" column="0" >
                  <widget class="QRadioButton" name="_changeSelected" >
                    <property name="text" >
                      <string>C&amp;hange selected vectors and matrices</string>
                    </property>
                    <property name="checked" >
                      <bool>true</bool>
                    </property>
                  </widget>
                </item>
                <item row="0" column="2" >
                  <spacer name="spacer3" >
                    <property name="sizeHint" >
                      <size>
                        <width>16</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Horizontal</enum>
                    </property>
                  </spacer>
                </item>
              </layout>
            </item>
            <item row="2" column="0" >
              <layout class="QGridLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <item row="0" column="2" >
                  <spacer name="spacer4" >
                    <property name="sizeHint" >
                      <size>
                        <width>16</width>
                        <height>20</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Horizontal</enum>
                    </property>
                  </spacer>
                </item>
                <item row="0" column="1" >
                  <widget class="QCheckBox" name="_duplicateDependents" >
                    <property name="text" >
                      <string>D&amp;uplicate dependents</string>
                    </property>
                    <property name="checked" >
                      <bool>true</bool>
                    </property>
                  </widget>
                </item>
                <item row="0" column="0" >
                  <widget class="QRadioButton" name="_duplicateSelected" >
                    <property name="text" >
                      <string>&amp;Duplicate selected vectors and matrices</string>
                    </property>
                  </widget>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <spacer name="Spacer15" >
              <property name="sizeHint" >
                <size>
                  <width>180</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="ChangeFileApply" >
              <property name="text" >
                <string>&amp;Apply</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>Apply new source file to selected vectors.</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="ChangeFileOK" >
              <property name="text" >
                <string>&amp;OK</string>
              </property>
              <property name="default" >
                <bool>true</bool>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>Apply new source file to selected vectors.</string>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="ChangeFileCancel" >
              <property name="text" >
                <string>&amp;Cancel</string>
              </property>
              <property name="whatsThis" stdset="0" >
                <string>Close dialog without applying further changes.</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <customwidgets>
    <customwidget>
      <class>KComboBox</class>
      <extends>QComboBox</extends>
      <header>kcombobox.h</header>
    </customwidget>
  </customwidgets>
  <tabstops>
    <tabstop>_allFromFile</tabstop>
    <tabstop>_files</tabstop>
    <tabstop>_clearFilter</tabstop>
    <tabstop>_filter</tabstop>
    <tabstop>ChangeFileCurveList</tabstop>
    <tabstop>ChangeFileClear</tabstop>
    <tabstop>ChangeFileSelectAll</tabstop>
    <tabstop>_dataFile</tabstop>
    <tabstop>_changeSelected</tabstop>
    <tabstop>_duplicateSelected</tabstop>
    <tabstop>_duplicateDependents</tabstop>
    <tabstop>ChangeFileApply</tabstop>
    <tabstop>ChangeFileCancel</tabstop>
  </tabstops>
  <includes>
    <include location="local" >kcombobox.h</include>
    <include location="local" >kurlrequester.h</include>
    <include location="local" >klineedit.h</include>
    <include location="local" >kpushbutton.h</include>
  </includes>
  <connections>
    <connection>
      <sender>ChangeFileCancel</sender>
      <signal>clicked()</signal>
      <receiver>KstChangeFileDialog</receiver>
      <slot>reject()</slot>
    </connection>
  </connections>
</ui>
